extra_data/tests/mockdata/base.py

```diff
@@ -153,5 +176,18 @@ def write_train_ids(f, path, N, first=10000, chunksize=16):
         Npad = N + chunksize - (N % chunksize)
     else:
         Npad = N
-    ds = f.create_dataset(path, (Npad,), 'u8', maxshape=(None,))
+
+    if format_version != '0.5':
+        # flag
+        ds = f.create_dataset('INDEX/flag', (Npad,), 'u4', maxshape=(None,))
```

Comment:
In the file I'm looking at, flag is a signed `int32` (`'i4'` in this syntax).

Reply:
oh, I thought that I checked that. updated.